cmake/branding/recolor_graphics.sh.in

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# shellcheck disable=SC2193
```

Comment:
Usually, instead of globally (by setting the `disable` at the top of the script), we add it just before the call that's known to fire the warning.  That way, we don't sacrifice potential bugs in code in other areas as a result of one, two or three exceptions.

https://github.com/koalaman/shellcheck/wiki/Ignore

Reply:
Added 👍 
Though, now that it's ignoring the error from before, it's running into other errors.